README.md

```diff
@@ -121,6 +121,7 @@ Templating variables:
 %c       Print story creation timestamp
 %updated Print story updated timestamp (if different from created)
 %j       Print full story as formatted JSON
+%gb      Print Git integration branch name
```

Comment:
@joshmfrankel I think we'll need to update the README here with the new formatting characters.

Reply:
Yup I missed it